src/olympia/users/admin.py

```diff
@@ -482,11 +482,12 @@ def known_ip_adresses(self, obj):
         # extra queries for each row of results), otherwise, look everywhere
         # we can.
         activity_ips = getattr(obj, 'activity_ips', None)
-        to_ipaddress = IPAddressBinaryField().to_python
         if activity_ips is not None:
-            # The GroupConcat value is a comma seperated string of the binary values.
-            ip_addresses = {to_ipaddress(ip) for ip in activity_ips.split(b',')}
+            # The GroupConcat value is a comma seperated string of the ip
+            # addresses (already converted to string thanks to INET6_NTOA).
+            ip_addresses = set(activity_ips.split(','))
```

Comment:
We still want the set to be `IPv4Address`|`IPv6Address` objects, so it's the same (apart from splitting on `','` instead)
```suggestion
            ip_addresses = {to_ipaddress(ip) for ip in activity_ips.split(',')}
```

Reply:
Do we ? AFAICT it's just used to format the string which is then returned and displayed in the results.

Reply:
Ah, because `IPLog` will return such objects, so we need the `set` contents to be consistent...